lib/Twig/Extensions/Extension/I18n.php

```diff
@@ -42,3 +42,41 @@ public function getName()
         return 'i18n';
     }
 }
+
+if(!function_exists('pgettext')){
+    /**
+     * Gets a translation checking the context.
+     *
+     * @param string $context
+     * @param string $original
+     *
+     * @return string
+     */
+    function pgettext($context, $original)
```

Comment:
Defining global functions without any prefixing looks wrong to me (or is this is not a polyfill for a function available in newer PHP version but not documented ?)